examples/subscription/creditCardSubscriptionExample.js

```diff
@@ -0,0 +1,139 @@
+
+// You will receive saved_token_id as part of the response when the initial card payment is accepted (will also available in the HTTP notification's JSON)
+// Refer to this docs: https://docs.midtrans.com/en/core-api/advanced-features?id=sample-3ds-authenticate-json-response-for-the-first-transaction
+// {
+//  ..
+//  "status_code": "200",
+//  "signature_key": "99bc6101abcda11c17a8d401747d1fdfb6085af88daf94c575aeaab79487b4b5534a7b5cc898314e5386c0def86f6f7676079bdb656362c5ccd2c9906b42e5a3",
+//  "saved_token_id_expired_at": "2021-12-31 07:00:00",
+//  "saved_token_id": "521111gmWqMegyejqCQmmopnCFRs1117",
+//  "payment_type": "credit_card"
+//  ..
+// }
+// Sample saved token id for testing purpose
+let savedTokenId = '521111gmWqMegyejqCQmmopnCFRs1117'
+
+
+// prepare CORE API parameter ( refer to: https://api-docs.midtrans.com/#create-subscription ) create subscription parameter example
+let parameter = {
+    "name": "MONTHLY_2021",
+    "amount": "14000",
+    "currency": "IDR",
+    "payment_type": "credit_card",
+    "token": savedTokenId,
+    "schedule": {
+        "interval": 1,
+        "interval_unit": "month",
+        "max_interval": 12,
+        "start_time": "2021-11-25 07:25:01 +0700"
+    },
+    "metadata": {
+        "description": "Recurring payment for A"
+    },
+    "customer_details": {
+        "first_name": "John",
+        "last_name": "Doe",
+        "email": "johndoe@email.com",
+        "phone": "+62812345678"
+    }
+};
+
+core.createSubscription(parameter)
+    .then((Response) => {
+        console.log('Response:', JSON.stringify(Response));
+    })
+    .catch((e) => {
+        console.log('Error occured:', e.message);
+    });
+// sample response is dictionary representation of API JSON response
+// {
+//     "id": "537d04c6-eef7-4c07-a65c-11c55f92c7ed",
+//     "name": "MONTHLY_2021",
+//     "amount": "14000",
+//     "currency": "IDR",
+//     "created_at": "2021-11-18 12:14:09",
+//     "schedule": {
+//     "interval": 1,
+//         "current_interval": 0,
+//         "max_interval": 12,
+//         "interval_unit": "month",
+//         "start_time": "2021-11-25 07:25:01",
+//         "next_execution_at": "2021-11-25 07:25:01"
+// },
+//     "status": "active",
+//     "token": "521111gmWqMegyejqCQmmopnCFRs1117",
+//     "payment_type": "credit_card",
+//     "transaction_ids": [],
+//     "metadata": {
+//     "description": "Recurring payment for A"
+// },
+//     "customer_details": {
+//     "email": "johndoe@email.com",
+//         "first_name": "John",
+//         "last_name": "Doe",
+//         "phone": "+62812345678"
+// }
+// }
+
+// Sample active subscription id for testing purpose
+let subscriptionId = "537d04c6-eef7-4c07-a65c-11c55f92c7ed"
+
+// get subscription by subscriptionId
+core.getSubscription(subscriptionId)
+    .then((Response) => {
+        console.log('Response:', JSON.stringify(Response));
+    })
+    .catch((e) => {
+        console.log('Error occured:', e.message);
+    });
```

Comment:
Have you try to run this code? every response will be printed confusingly as `Response` without proper difference between which function call's response. Let's not be lazy, you should print it properly e.g `getSubscription response:` & `getSubscription error  message:`

Even the python reference is not this lazy, follow the reference as the minimum standard, you work shouldn't be worse than reference.